### ISSUE TITLE
Kevin Frey, Q4 interviews

### ISSUE THREAD

**Issue by VanCardui** [SingularityNet-Ambassador-Program/AI-Ethics-Workgroup#68]
**Interviewees:** Strangers, 1 family member that has no interest in AI, a couple of nurses and medical professionals, a phlebotomist, a tattoo artist, etc
**Number of people:** 10
**Interview hours:** 5
**Fee:** $250

**Comment by TheFreysDefi** [comment]
We are conducting interviews today

**Comment by VanCardui** [comment]
Did 7 - all done, with help from Advanceameyaw
Will check if Esther has created folder, and if yes, Kevin will upload

**Comment by TheFreysDefi** [comment]
Thank you Vanni!

**Comment by TheFreysDefi** [comment]
I did pay for everyone's travel to get to the podcast studio and provided food due to the timing of the interview because it is just good manners, they were hungry and waiting to do this to help me. 

I also paid for the podcast studio which is unnecessary to reimburse me for since I did it on my own without prior approval. It just made things easier for me. That was 500 Cedis. A 300 Cedis deposit and 200 Cedis paid the day of the interviews. The exchange rate on that is $33.

If the group approves the $10 for expenses per interviewee it would be helpful to offset my expenses.

Thank you all.

**Comment by VanCardui** [comment]
Still checking validity of interviews - several are a bit short
Expenses will be paid regardless

**Comment by TheFreysDefi** [comment]
Yeah it’s March Vannie. I did these in December. I’ve moved past this.

Thank you.

**Comment by VanCardui** [comment]
5 interviews submitted
Validity checked. Costs as per https://github.com/SingularityNet-Ambassador-Program/AI-Ethics-Workgroup/issues/117

- Wgq-KF-001 - 26 mins, so $25, plus £10 expenses - total **$35**
- Wgq4-KF-002 - short (22 mins) so $10, plus $10 expenses - total **$20**
- Wgq4-KF-004 - short (21 mins) so $10, plus $10 expenses - total **$20**
- Wgq4-KF-003 and Wgq4-KF-005 (2 different consent form identifier codes, same interview): short (8 mins) so $5, plus $20 expenses (2 interviewees) - total **$25**
- Amo and Sylvanus - no consent forms. $0. Plus $20 expenses (2 interviewees) - total **$20**

Overall total $120
